statsmodels/robust/norms.py

```diff
@@ -463,6 +530,16 @@ def rho(self, z):
         r"""
         The robust criterion function for Andrew's wave.
 
+        .. math::
+
+            \rho: \mathbb{R}^n \to \mathbb{R}^n
+
+            \rho(z) = \begin{cases}
+                          a^2 (1 - \cos(\frac{z}{a})), & \text{if }
+                          \lvert z \rvert \le a\pi \\
+                          2a, & \text{if } \lvert z \rvert > a\pi
+                      \end{cases}
+
```

Comment:
This feel redundant given then math below.  We should consolidate these definitions with the formula explanation contained in each, so that there aren't two explanations. Sometimes the formula is python math, sometimes it is LaTeX. I'm happy if you put your latex in place of the which ever is there.

Reply:
Ok. Thank you for the feedback @bashtage. I understand  - so ideally there would always be one formula (either latex or python math). I would therefore refactor  all docstrings in this file to have the LaTex formulas in the "Returns" statement to consolidate the information of *what the function does* in there.

This would look like this:
```python
def rho(self, z):
        r"""
        The robust criterion function for Andrew's wave.

        Parameters
        ----------
        z : array_like
            1d array

        Returns
        -------
        rho : ndarray
            The elements of rho are defined as:

        .. math::

            \rho: \mathbb{R}^n \to \mathbb{R}^n

            \rho(z) = \begin{cases}
                            a^2 (1 - \cos(\frac{z}{a})), & \text{if }
                            \lvert z \rvert \le a\pi \\
                            2a, & \text{if } \lvert z \rvert > a\pi
                        \end{cases}
        """
```

This way there would be one single place to look for this information. Is this how you imagined it?